pkg/network/render.go

```diff
@@ -385,6 +388,59 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	return nil
 }
 
+func isClusterNetworkChangeSafe(prev, next *operv1.NetworkSpec) bool {
+
```

Comment:
I think it would be good for clarity to remove the first part of the following statement from `isNetworkChangeSafe`:
```
if reflect.DeepEqual(prev.ClusterNetwork, next.ClusterNetwork) && reflect.DeepEqual(prev.ServiceNetwork, next.ServiceNetwork) {
		return nil
}
```
and move `if reflect.DeepEqual(prev.ClusterNetwork, next.ClusterNetwork) {return nil}`  to `isClusterNetworkChangeSafe`